portfolio/src/main/java/com/google/sps/data/BlogComment.java

```diff
@@ -7,10 +7,12 @@
 
   private final Date postDate;
   private final String text; 
+  private final String imageURL;
```

Comment:
Should we mark it nullable since image can be empty in which case this field will be null.

Reply:
Okay sure, I will mark it as **@Nullable**